drivers/sensor/max1726x/max1726x.c

```diff
@@ -62,16 +63,63 @@ static int max1726x_reg_write(const struct device *dev, uint8_t reg_addr,
 		     cfg->i2c_addr);
 }
 
+/**
+ * @brief Convert current in MAX1726X units to milliamps
+ *
+ * @param rsense_mohms Value of Rsense in milliohms
+ * @param val Value to convert (taken from a MAX1726X register)
+ * @return corresponding value in milliamps
+ */
+static int current_to_ma(unsigned int rsense_mohms, int16_t val)
+{
+	return (val * 1.5625) / rsense_mohms;
```

Comment:
constants should be defined, non-fixed width `int` types do not sound appropriate here.

Reply:
Modified to uint16_t